client/assets/colors.scss

```diff
@@ -1,5 +1,7 @@
 $colors: (
-  yellow: #FFA300,
-  light: #FFF,
+  yellow: #ffa300,
+  light: #fff,
   dark: #000,
-)
+  bonjour: #e0dcdc,
```

Comment:
is bonjour a color ? you should hive this a specific color name

Reply:
Apparently, it is 😃  . I followed http://chir.ag/projects/name-that-color/ as recommended in our wiki to get convert `#e0dcdc` and that was the matched result.